manifests/golang.yml

```diff
@@ -468,6 +468,18 @@ tests/:
     test_otel_sdk_interoperability.py: missing_feature
     test_otel_span_with_baggage.py:
       Test_Otel_Span_With_Baggage: missing_feature
+    test_parametric_endpoints.py:
+      Test_Parametric_DDSpan_Add_Link: missing_feature (add_link endpoint is not implemented)
+      Test_Parametric_DDSpan_Set_Resource: missing_feature (set_resource endpoint is not implemented)
+      Test_Parametric_DDTrace_Baggage: missing_feature (baggage is not supported)
+      Test_Parametric_DDTrace_Crash: missing_feature (crash endpoint is not implemented)
+      Test_Parametric_DDTrace_Current_Span: missing_feature (current span endpoint is not defined)
```

Comment:
`test_library.current_span()` seems to return the current active span on the tracer, but dd-trace-go doesn't have this concept, spans are stored in local context objects.
Do we still want to say the endpoint isn't implemented, or do we want to say this feature isn't supported / compatible?

Reply:
Ya that's a good point. We should say this feature isn't supported. This way someone doesn't go in and try to add the endpoint in the future.

Reply:
> In python we use a web framework that stores spans in the context of a request. So a span is only active/current in the context of the request it was initialized in.
Yes, Go is the same.

> Is this a flow we can add/mock in the go parametric app.
No, not with the current functionality.

How do you think we should label this test then? 🤔